README.md

```diff
@@ -26,6 +26,9 @@ Options:
   --cmd                                                               [required]
   --started-by
   --env
+  --fargate
```

Comment:
Rather than have a flag explicitly for fargate, it would be better to have a `--launch-type` flag that defaults to EC2 and can be set to Fargate optionally.

In addition, it might make sense to have a networking mode flag to set it explicitly to awsvpc (which can be used with the Fargate or EC2 launch types).

Reply:
Changed it to the launch-type parameter, allowing `FARGATE`, `EC2` and `EXTERNAL`.

Networking mode is set on the task definition, not on the runtask command so no need to add an option to this tool. I've added validation to ensure network options are set when the taks specifies the `awsvpc` networkMode.